eessi/testsuite/__init__.py

```diff
@@ -0,0 +1,16 @@
+# If this is an installed package, setuptools_scm should have written the file _version.py in the current directory
+try:
+    from ._version import __version__
+except ImportError:
+    # Fallback for when the package is not installed, but git cloned. Note that this requires setuptools_scm to be
+    # available as a runtime dependency
+    try:
+        from setuptools_scm import get_version
+        # Using a relative path for relative_to doesn't work, because it will be relative to the current working
+        # directory (which could be anywhere)
+        # __file__ is the location of this init file (a full path), and this gives us a predictable path to the root
+        # (namely: two levels up)
+        # Note that if we ever move this __init__ file relative to the root of the git tree, we'll need to adjust this
+        __version__ = get_version(root='../..', relative_to=__file__)
+    except ImportError:
+        __version__ = "0.0.0"  # fallback version if setuptools_scm is not available
```

Comment:
Like I mentioned in the other PR, in general I'm not a big fan of `setuptools_scm`, I've been bitten by it one too many times already.

How does it determine the version exactly? Based on Git metadata? What if that's not available, because someone downloaded a tarball that doesn't contain that metadata?